firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -24,7 +25,54 @@ void moveMotor(int motor[], int steps, int dir) {
 
 }
 
-void disableMotors() {
+void moveSpaces(int motor[], int spaces, int dir)
+{
+  int i;
+
+  // How many steps per space
+  float numSteps = spaces * stepsPerSpace;
+
+  digitalWrite(MOTOR_SLEEP, HIGH);
+  digitalWrite(MOTOR_RESET, HIGH);
+  digitalWrite(MOTOR_ENABLE, LOW);
+
+  digitalWrite(motor[1], dir);
+
+  for(i = 0; i < numSteps; i++);
+  {
+    digitalWrite(motor[0], LOW);
+    delay(1);  // 1 milliSecond
+    digitalWrite(motor[0], HIGH);
+  }
+
+}
+
+void moveDiaganol(int motor[], int dir, int spaces)
+{
+  int i;
+
+  // similary subject to change as above function dictation, same number as above though
+  float numSteps = spaces * stepsPerSquare;
```

Comment:
should this be stepsPerSpace?

Reply:
I believe this will also raise an error with arduino, since stepsPerSquare no longer exists.